src/main/java/game/Gamestate.java

```diff
@@ -12,9 +12,11 @@
 public final class Gamestate {
 	private int state;
 	private int mode;
-	private long pauseTime;
-	private long restartTime;
+	private long screenSwitchTime;
```

Comment:
het leek mij duidelijk dat dat de tijd is tussen de verschillende schermen. anders switchen de schermen sneller dan je kan zien als je de knop ingedrukt houdt. (dit was een makkelijkere oplossing dan te onthouden welke knop er al ingedrukt was. dan zou de launcher ipv van alleen input ook een released lijst mee moeten geven of overal onthouden welke knop er ingedrukt was.)